### PR TITLE
Cleanup iOS.

### LINKED ISSUE #42: iOS: disable/hide the reviews
image

### LINKED ISSUE #43: iOS: remove all ads

### LINKED ISSUE #44: iOS: disable/hide the guide layer and a button from the main menu
image

### LINKED ISSUE #45: iOS: disable/hide paid subscriptions
image

### LINKED ISSUE #69: iOS: disable/hide the traffic layer
image
image

### LINKED ISSUE #70: iOS: remove three onboarding screens
See #55

### LINKED ISSUE #73: iOS: remove Facebook SDK

### LINKED ISSUE #79: iOS: disable/hide the traffic button in navigation mode
See #77

### PR DESCRIPTION
**Pull request by vng**
Fixes #42
Fixes #43
Fixes #44
Fixes #45
Fixes #69
Fixes #70
Fixes #73
Fixes #79

### CODE REVIEW

**Review by rtsisyk — CHANGES_REQUESTED** [review]
1. Let's keep crashlytics, we need crash reports.
2. Please also remove Google && Facebook from Settings->OpenStreetMap Login

File: tools/autobuild/detect_qmake.sh [hunk 1]
```diff
@@ -6,13 +6,7 @@ set -e -u
...
 # Add your path into this array
 KNOWN_QMAKE_PATHS=( \
-  /Developer/Tools/qmake \
-  /usr/local/opt/qt5/bin/qmake \
-  /usr/local/opt/qt@5.?/bin/qmake \
-  ~/Developer/Qt/5.?/clang_64/bin/qmake \
-  ~/Qt/5.?/clang_64/bin/qmake \
-  ~/Qt5.?.0/5.?/clang_64/bin/qmake \
-  /cygdrive/c/Qt/5.?/msvc2013_64/bin/qmake.exe \
+  ~/Developer/Qt/5.8/clang_64/bin/qmake \
```
[the remark this answers is not in the document]
Comment (vng) [reply tools/autobuild/detect_qmake.sh:9]: Looks like this detect_qmake.sh is not needed any more .. I also don't have Qt/5.8

[the remark this answers is not in the document]
Comment (rtsisyk) [reply tools/autobuild/detect_qmake.sh:9]: Let's remove this script then.

**Comment by rtsisyk** [comment]
The last, but not least: please accept [DCO](https://github.com/omapsapp/omapsapp/blob/master/docs/DCO.md) and add ["Signed-off-by" to each commit](https://github.com/omapsapp/omapsapp/blob/master/docs/CONTRIBUTING.md#pull-requests):

1. git rebase -i HEAD~[NUMBER OF COMMITS]
2. "reword"
3. append "Signed-of-by: Your real name <email>"

**Review by rtsisyk — APPROVED** [review]

**Review by rtsisyk — APPROVED** [review]

**Pull request merged**